app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -129,9 +129,14 @@ def save_challenge_in_session
 
     def process_valid_webauthn(form)
       create_user_event(:webauthn_key_added)
+      mfa_user = MfaContext.new(current_user)
+      analytics.user_registration_mfa_webauthn_added(
+        platform_authenticator: form.platform_authenticator?,
+        enabled_mfa_methods_count: mfa_user.enabled_mfa_methods_count + 1,
```

Comment:
QQ I'm seeing all these `+ 1` in the various methods, is it possible to move this event to after the MFA is created? Or add a `.reload` somewhere? It seems suspicious to me to add this when we know the DB is about to be changed (or has just changed) in successful case